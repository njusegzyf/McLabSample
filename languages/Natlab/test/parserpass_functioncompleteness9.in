function f(x)
%%helpcomment
y = x;
end