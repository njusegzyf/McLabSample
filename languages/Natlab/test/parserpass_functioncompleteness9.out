1 1
4 3
function [] = f(x)
%%helpcomment
y = x;
end
